src/core/dynamic_batch_scheduler.cc

```diff
@@ -37,36 +37,40 @@
 #include "src/core/server.h"
 #include "triton/common/nvtx.h"
 
-namespace nvidia { namespace inferenceserver {
-
-bool
-IsStaleState(Payload::State payload_state)
-{
-  return (
-      (payload_state == Payload::State::EXECUTING) ||
-      (payload_state == Payload::State::RELEASED));
+namespace nvidia {
+namespace inferenceserver {
```

Comment:
Why did you change this formatting and other formatting within the file? This need to be reverted.